services/app/src/components/chessBoard/ChessBoard.jsx

```diff
@@ -1,54 +1,30 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
```

Comment:
I can see very similar changes in the other PR https://github.com/ZenitechSoftware/planning-chess/pull/46/files.
I assume, both PRs depend on some common functionality, that has not been merged to `main` just yet. If this is a case, I would suggest to re-raise this PR against the other feature branch instead of main. It will be easier to review changes that are related to this PR only.

Reply:
This issue comes before header styling; therefore, my understanding is that we should be merged into `main` first.

Reply:
If current pull request (A) needs to be merged first but contains changes that are used by other pull request (B), the flow should be the following:
* Open pull request to merge A to main
* Open pull request to merge B to A

This way it will be easier to track changes that are specific to pull request A and pull request B. Github should take care of reopening pull request B against main if A gets merged first.

Let's not reopen `header` pull request this time, but let's try to follow this flow the next time simillar situation occurs.